Sources/FileManager+Testable.swift

```diff
@@ -31,4 +31,40 @@ extension FileManager {
             }
         }
     }
+    
+    /// Removes the Adobe-specific directory within the app's data storage (persistence). This method attempts to locate and delete the 
+    /// `com.adobe.aep.datastore` directory either in the specified app group's container directory or in the default library directory 
+    /// if no app group is provided.
+    ///
+    /// - Parameter appGroup: An optional `String` representing the app group identifier. If provided, the method will look for the Adobe directory within the shared container for that app group. If `nil`, the method will search for the directory in the user's library directory across all domains the app has access to.
+    ///
+    /// - Requires: Before calling this method, ensure that the caller has the appropriate permissions to access and modify the file system, especially if working with app group directories.
+    func removeAdobeDirectory(appGroup: String?) {
```

Comment:
i like your idea of giving this param a default value.  

for future flexibility, it might also be good to allow the "directory" to be passed in, and use the `com.adobe.aep.datastore` value as default.

i propose the following:
```
func removeAdobeCacheDirectory(_ directoryName: String? = "com.adobe.aep.datastore", 
                               with appGroup: String? = nil) {
    ...
}
```

this change also allows for the most common call to this method, eliminating all the adobe related cache files with no app group, to be as simple as possible:
```
FileManager.default.removeAdobeCacheDirectory()
```

Reply:
my two cents is that since we expect the common usage to be clearing the whole folder, leaving the param optional is better.  but not a hill i'm going to die on :) 

also wondering if using `in appGroup:` is better than using my previously suggested `with appGroup:`... 🤔